staff/kapil-siva/course-1/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>NYSL</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="second" href="index1.html" />
```

Comment:
this <link> here with `index1.html` is not correct and it should be used only to link things like styles and so on